charts/nexus-iq/templates/deployment.yaml

```diff
@@ -28,6 +28,23 @@ spec:
       serviceAccountName: {{ include "iqserver.serviceAccountName" . }}
       securityContext:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      {{- if .Values.initContainers.enabled }}
+      initContainers:
+        - name: chown-vols
+          image: "busybox:latest"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          securityContext:
+            runAsUser: 0
+            allowPrivilegeEscalation: true
+          command: ['sh', '-c', 'chown -R {{ .Values.initContainers.updateUser }}:{{ .Values.initContainers.updateGroup }} /var/log/nexus-iq-server {{ .Values.configYaml.sonatypeWork }}']          
+          volumeMounts:
+            - mountPath: {{ .Values.configYaml.sonatypeWork }}
+              name: nxiq-pv-data
+            - mountPath: /var/log/nexus-iq-server
+              name: nxiq-pv-log
+            - mountPath: /etc/nexus-iq-server
+              name: config-volume
+      {{- end }}
```

Comment:
it would be good to add a unit test that flips enabled: true, and asserts about some of the values found here.

Reply:
I can resubmit the PR with name change and if you could help with adding unit test, @jflinchbaugh would be much appreciated.

Reply:
Oh, you didn't need to close the PR. Since it's a branch in our own repo, you could continue to commit to it, and this PR would reflect your new changes. I'll be happy to introduce the tests on your branch. When I see the new PR, i'll hop in and commit the tests as well, so we can see them together with your updates.